mithril-aggregator/src/artifact_builder/cardano_transactions.rs

```diff
@@ -44,10 +44,10 @@ impl ArtifactBuilder<CardanoDbBeacon, CardanoTransactionsSnapshot>
             .with_context(|| {
                 format!(
                     "Can not compute CardanoTransactionsCommitment artifact for signed_entity: {:?}",
```

Comment:
Not a consequence of this PR
```suggestion
                    "Can not compute CardanoTransactions artifact for signed_entity: {:?}",
```